src/Spice86/Models/Debugging/LinearMemoryAddress.cs

```diff
@@ -0,0 +1,27 @@
+﻿namespace Spice86.Models.Debugging;
+
+using Spice86.Core.Emulator.Memory;
+
+using System.ComponentModel.DataAnnotations;
+
+public readonly record struct LinearMemoryAddress {
+    private readonly string? _sourceInput;
+    public LinearMemoryAddress([Range(0, A20Gate.EndOfHighMemoryArea)] uint address, string? sourceInput = null) {
+        Address = address;
+    }
```

Comment:
Did you mean
```suggestion
        Address = address;
        _sourceInput = sourceInput;
    }
```
? Or it the whole sourceinput thing obsolete?

Reply:
Fixed, _sourceInput is not obsolete. Not yet, anyway.